### PR TITLE
xsiam-radware-appwall-mapping-ciac-7903

### PR DESCRIPTION
**Pull request by cweltPA**
## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: [CIAC-7903](https://jira-dc.paloaltonetworks.com/browse/CIAC-7903)

## Description
This PR contains mappings for the Radware AppWall WAF cloud services alerts. 

## Must have
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

File: Packs/RadwareCloudServices/README.md [hunk 1]
```diff
@@ -1,15 +1,27 @@
...
+This pack includes Cortex XSIAM content.  
+It supports the following Radware Cloud Services event types: 
+- [Radware Cloud WAAP Service Access Logs](https://support.radware.com/app/answers/answer_view/a_id/1018351/~/cloud-waf-service-access-log-integration-guide).
+  - Access logs provide detailed data regarding client access to applications protected by Cloud WAAP. 
+- [Radware AppWall Cloud WAF Security Event Logs](https://support.radware.com/app/answers/answer_view/a_id/1018635/~/reviewing-security-events).
+  - A security event is generated whenever Cloud WAF detects an attack, when an ongoing attack
+is still active, or when an ongoing attack status has changed. The generated security event
+includes the information relevant to the specific attack or security breach.
+
+See the documentation bellow for the configuration required for each event type you wish to collect into Cortex XSIAM.
```
Comment (ShirleyDenkberg) [inline Packs/RadwareCloudServices/README.md:13]:
```suggestion
See the documentation below for the configuration required for each event type to collect into Cortex XSIAM.  
```

File: Packs/RadwareCloudServices/README.md [hunk 2]
```diff
@@ -43,6 +55,73 @@ For more information, refer to the official Radware [documentation](https://supp
 5. Creating a new HTTP Log Collector will allow you to generate a unique token. Save it since it will be used later.
 6. Click the 3 dots next to the newly created instance and copy the API URL. It will also be used later.
 
-For more information, see this [doc](https://docs-cortex.paloaltonetworks.com/r/Cortex-XSIAM/Cortex-XSIAM-Administrator-Guide/Ingest-Generic-Logs-from-Amazon-S3).
+For additional information, see this [doc](https://docs-cortex.paloaltonetworks.com/r/Cortex-XSIAM/Cortex-XSIAM-Administrator-Guide/Ingest-Generic-Logs-from-Amazon-S3).
+
+## Collect Radware AppWall Cloud WAF Security Events 
+
+In order to collect AppWall Cloud WAF Security events, you would need to deploy a log-collection solution such as [Logstash](https://www.elastic.co/logstash) that can interact with [Amazon SQS](https://aws.amazon.com/sqs/), retrieve your queued event messages from Radware Cloud and forward them via Syslog to Cortex XSIAM.
```
Comment (ShirleyDenkberg) [inline Packs/RadwareCloudServices/README.md:62]:
```suggestion
In order to collect AppWall Cloud WAF Security events, you need to deploy a log-collection solution such as [Logstash](https://www.elastic.co/logstash) that can interact with [Amazon SQS](https://aws.amazon.com/sqs/), retrieve your queued event messages from Radware Cloud and forward them via Syslog to Cortex XSIAM. 
```

File: Packs/RadwareCloudServices/README.md [hunk 3]
```diff
@@ -43,6 +55,73 @@ For more information, refer to the official Radware [documentation](https://supp
...
+For additional information, see this [doc](https://docs-cortex.paloaltonetworks.com/r/Cortex-XSIAM/Cortex-XSIAM-Administrator-Guide/Ingest-Generic-Logs-from-Amazon-S3).
+
+## Collect Radware AppWall Cloud WAF Security Events 
+
+In order to collect AppWall Cloud WAF Security events, you would need to deploy a log-collection solution such as [Logstash](https://www.elastic.co/logstash) that can interact with [Amazon SQS](https://aws.amazon.com/sqs/), retrieve your queued event messages from Radware Cloud and forward them via Syslog to Cortex XSIAM. 
+
+Radware recommends using [Logstash](https://www.elastic.co/logstash) as the log-collection solution, however, any other third-party log-collection solution can be used as long as it has an interface with [Amazon SQS](https://aws.amazon.com/sqs/) and supports syslog forwarding. 
+
+The following steps demonstrate the configuration steps with Logstash used as the log-collection solution, under the assumption that Logstash has already been installed & deployed on your environment.
```
Comment (ShirleyDenkberg) [inline Packs/RadwareCloudServices/README.md:66]:
```suggestion
The following steps demonstrate the configuration steps with Logstash used as the log-collection solution, under the assumption that Logstash has already been installed and deployed on your environment. 
```

File: Packs/RadwareCloudServices/README.md [hunk 4]
```diff
@@ -43,6 +55,73 @@ For more information, refer to the official Radware [documentation](https://supp
...
+
+In order to collect AppWall Cloud WAF Security events, you would need to deploy a log-collection solution such as [Logstash](https://www.elastic.co/logstash) that can interact with [Amazon SQS](https://aws.amazon.com/sqs/), retrieve your queued event messages from Radware Cloud and forward them via Syslog to Cortex XSIAM. 
+
+Radware recommends using [Logstash](https://www.elastic.co/logstash) as the log-collection solution, however, any other third-party log-collection solution can be used as long as it has an interface with [Amazon SQS](https://aws.amazon.com/sqs/) and supports syslog forwarding. 
+
+The following steps demonstrate the configuration steps with Logstash used as the log-collection solution, under the assumption that Logstash has already been installed & deployed on your environment. 
+
+### Obtain Amazon SQS Credentials 
+**On Radware Cloud WAF Portal**
+1. Connect to your Radware account on Radware [Cloud WAF Portal](https://portal.cwp.radwarecloud.com/#/login).
```
Comment (ShirleyDenkberg) [inline Packs/RadwareCloudServices/README.md:70]:
```suggestion
1. Connect to your Radware account on the Radware [Cloud WAF Portal](https://portal.cwp.radwarecloud.com/#/login).
```

File: Packs/RadwareCloudServices/README.md [hunk 5]
```diff
@@ -43,6 +55,73 @@ For more information, refer to the official Radware [documentation](https://supp
...
+2. Navigate to **Account** &rarr; **Account Settings**. 
+3. Click **Download SIEM Configuration** to download a configuration file which includes the details of the SQS event queues & your credentials for accessing them. This file has the name convention of *siemConfigFetchConfig_\<ID\>.txt*. Use this file in the next section when configuring Logstash (or any other log-collection solution you wish to use).
```
Comment (ShirleyDenkberg) [inline Packs/RadwareCloudServices/README.md:72]:
```suggestion
3. Click **Download SIEM Configuration** to download a configuration file which includes the details of the SQS event queues and your credentials for accessing them. This file has the name convention of *siemConfigFetchConfig_\<ID\>.txt*. Use this file in the next section when configuring Logstash (or any other log-collection solution you want to use).
```

File: Packs/RadwareCloudServices/README.md [hunk 6]
```diff
@@ -43,6 +55,73 @@ For more information, refer to the official Radware [documentation](https://supp
...
+
+### Configure Logstash  
+The downloaded SIEM configuration file that was downloaded from Radware portal (see the previous section) already contains a predefined SQS input plugin for retrieving events from Amazon SQS.
+You would need to update this file to include a syslog *output* plugin that would forward the retrieved event messages to your Cortex XSIAM Broker VM via syslog.
```
Comment (ShirleyDenkberg) [inline Packs/RadwareCloudServices/README.md:76]:
```suggestion
You need to update this file to include a syslog *output* plugin that would forward the retrieved event messages to your Cortex XSIAM Broker VM via syslog. 
```

File: Packs/RadwareCloudServices/README.md [hunk 7]
```diff
@@ -43,6 +55,73 @@ For more information, refer to the official Radware [documentation](https://supp
...
+**On Radware Cloud WAF Portal**
+1. Connect to your Radware account on Radware [Cloud WAF Portal](https://portal.cwp.radwarecloud.com/#/login).
+2. Navigate to **Account** &rarr; **Account Settings**. 
+3. Click **Download SIEM Configuration** to download a configuration file which includes the details of the SQS event queues & your credentials for accessing them. This file has the name convention of *siemConfigFetchConfig_\<ID\>.txt*. Use this file in the next section when configuring Logstash (or any other log-collection solution you wish to use).
+
+### Configure Logstash  
+The downloaded SIEM configuration file that was downloaded from Radware portal (see the previous section) already contains a predefined SQS input plugin for retrieving events from Amazon SQS.
+You would need to update this file to include a syslog *output* plugin that would forward the retrieved event messages to your Cortex XSIAM Broker VM via syslog. 
+
+1. Open the *siemConfigFetchConfig_\<ID\>.txt* SIEM configuration file that was downloaded from Radware portal in the previous section.
```
Comment (ShirleyDenkberg) [inline Packs/RadwareCloudServices/README.md:78]:
```suggestion
1. Open the *siemConfigFetchConfig_\<ID\>.txt* SIEM configuration file that was downloaded from the Radware portal in the previous section.
```

File: Packs/RadwareCloudServices/README.md [hunk 8]
```diff
@@ -43,6 +55,73 @@ For more information, refer to the official Radware [documentation](https://supp
...
+2. Define a [Syslog output plugin](https://www.elastic.co/guide/en/logstash/current/plugins-outputs-syslog.html) entry with the following properties:
+   | Property     | Value    
+   | :---          | :---    
+   | `host`        | Enter The IP address of the Cortex XSIAM Broker VM syslog server.             
+   | `port`        | Enter the syslog service port number that Cortex XSIAM Broker VM should listen on for receiving Radware AppWall security events that would be forwarded from Logstash. 
+   | `rfc`         | Enter **rfc5424**
+   | `appname`     | Enter a meaningful application name for the syslog message. For e.g. **RadwareAppWall**.
```
Comment (ShirleyDenkberg) [inline Packs/RadwareCloudServices/README.md:85]:
```suggestion
   | `appname`     | Enter a meaningful application name for the syslog message. For e.g., **RadwareAppWall**.
```

**Comment by ShirleyDenkberg** [comment]
@eepstain Doc review completed.

**Review by eepstain — APPROVED** [review]

**Pull request merged**